src/hotspot/cpu/ppc/stubGenerator_ppc.cpp

```diff
@@ -4501,6 +4501,74 @@ class StubGenerator: public StubCodeGenerator {
 
 #endif // VM_LITTLE_ENDIAN
 
+  RuntimeStub* generate_cont_doYield() {
+    if (!Continuations::enabled()) return nullptr;
+    Unimplemented();
+    return nullptr;
+  }
+
+  address generate_cont_thaw(bool return_barrier, bool exception) {
+    if (!Continuations::enabled()) return nullptr;
+    Unimplemented();
+    return nullptr;
+  }
+
+  address generate_cont_thaw() {
+    if (!Continuations::enabled()) return nullptr;
+    Unimplemented();
+    return nullptr;
+  }
+
+  address generate_cont_returnBarrier() {
+    if (!Continuations::enabled()) return nullptr;
+    Unimplemented();
+    return nullptr;
+  }
+
+  address generate_cont_returnBarrier_exception() {
+    if (!Continuations::enabled()) return nullptr;
+    Unimplemented();
+    return nullptr;
+  }
+
+#if INCLUDE_JFR
+
+  static RuntimeStub* generate_jfr_stub(const char* name, address entrypoint) {
+    int insts_size = 512;
+    int locs_size = 64;
+    CodeBuffer code(name, insts_size, locs_size);
+    OopMapSet* oop_maps = new OopMapSet();
+    MacroAssembler* masm = new MacroAssembler(&code);
+    MacroAssembler* _masm = masm;
+
+    address start = __ pc();
+    __ unimplemented(FILE_AND_LINE);
```

Comment:
If this is going to call unimplemented(), you might have to not define JFR_HAVE_INTRINSICS for PPC.

Reply:
Will JFR not work otherwise even if the preview is disabled?

Reply:
See https://bugs.openjdk.java.net/browse/JDK-8286480 I was discussing this with Markus earlier.  He's going to fix this and then it's fine.

Reply:
Hi @reinrich, I am about to check in https://github.com/openjdk/jdk/pull/8631, hopefully making this a bit easier on you.